src/main/java/uk/gov/companieshouse/registeredemailaddressapi/controller/RegisteredEmailAddressController.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.companieshouse.registeredemailaddressapi.controller;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class RegisteredEmailAddressController {
+
+    @GetMapping("/registered-email-address/test")
```

Comment:
Can we map this to /healthcheck instead of /test?

This will enable us to adhere to the CH healthcheck guidelines - https://github.com/companieshouse/styleguides/blob/main/health_check.md